content/community/newsletter-archive.md

```diff
@@ -7,6 +7,7 @@ online: false
 
 2021
 
+- [September 2021](https://newsletters.getty.edu/t/t-636BF3B756B3461C2540EF23F30FEDED): New Feature Launch: Multi-level Subsections, Quire Design Guide for YAML & Markdown, Introducing Pull Request Templates, Tilt West Journal
```

Comment:
@dimrkh. Great! Can you add MuseumNext to this list? Just to follow the same pattern as the previous months (see July newsletter).